task3.1/task3.1/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task3._1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+           var  a = Console.ReadLine();
+            int a1 = Convert.ToInt32(a);
```

Comment:
Программа ожидает ввода данных, но пользователь не понимает что от него хотят. Перед вводом нужно печатать сообщения, чтобы было понятно, что требуется сделать. 
Запоминать введенную пользователем строку в отдельной переменной не имеет смысла - вы ее используете только один раз. 
Старайтесь избегать использования класса Convert без особой необходимости - пользуйтесь методами имеющихся типов.
Выбор типа данных неверный. Почему числа должны быть целыми? Вычисления в целых числах имеет свою специфику. Например, при делении целого числа на целое дробная часть частного отбрасывается.
В результате ваша программа вычисляет среднее арифметическое чисел 1, 1 и 2 и получает 1.
```suggestion
           Console.WriteLine("Введите первое число:");
            var a1 = double.Parse(Console.ReadLine());
```